clients/src/main/java/org/apache/kafka/clients/consumer/internals/Fetcher.java

```diff
@@ -1043,6 +1142,10 @@ private PartitionRecords parseCompletedFetch(CompletedFetch completedFetch) {
     @Override
     public void onAssignment(Set<TopicPartition> assignment) {
         sensors.updatePartitionLagAndLeadSensors(assignment);
+        /*
+         * clear the buffered data for paused partitions on new assignment
```

Comment:
We usually only add comment inside method using `//` rather than /* */`. Also, the method name here is clear enough and the comment seems unnecessary.